SchedulerBot/SchedulerBot.Database.Entities/ScheduledMessageDetailsServiceUrl.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace SchedulerBot.Database.Entities
+{
+	/// <summary>
+	/// Enables many-to-many relationship between <see cref="ScheduledMessageDetails"/> and <see cref="ServiceUrl"/>.
+	/// </summary>
+	public class ScheduledMessageDetailsServiceUrl
+	{
+		/// <summary>
+		/// Gets or sets the scheduled message details identifier.
+		/// </summary>
+		public Guid DetailsId { get; set; }
```

Comment:
`DetailsId` possibly rename

Reply:
Hm, not sure I am following.

It specifies the ID of ScheduledMessageDetails, which is pretty much `DetailsId`. Another thing is that the `DetailId` is actually also the ID of ScheduledMessage, i.e. it is the same that `MessageId`.

Is that what you've meant?

Reply:
Idea was to give a name `ScheduledMessageDetailId`

Reply:
@pryabov We could do that, but I think then we would have to rename `Details` property to `ScheduledMessageDetails` for consistency.

Do you think we should do this?